### PR TITLE
Adding timechart

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/timechart/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (scopatz) [inline recipes/timechart/meta.yaml]: please remove all comments

File: recipes/timechart/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pydm >=1.6.0
```
Comment (scopatz) [inline recipes/timechart/meta.yaml]: Is this needed at pip install time?

File: recipes/timechart/meta.yaml (hunk 1 above)
Comment (hhslepicka) [reply recipes/timechart/meta.yaml]: Addressed via https://github.com/conda-forge/staged-recipes/pull/11065/commits/72fce75c4efaed13c114078f7d3fe2a0ae0c4860

File: recipes/timechart/meta.yaml (hunk 2 above)
Comment (hhslepicka) [reply recipes/timechart/meta.yaml]: Addressed via https://github.com/conda-forge/staged-recipes/pull/11065/commits/48331451a38ce7e3cb5a259b6583f140e08379e1

**Comment by hhslepicka** [comment]
Windows is failing due to the entry_point script.

**Comment by scopatz** [comment]
you can add entry points to the build section if you have them

**Comment by hhslepicka** [comment]
@scopatz do you have an example?

**Comment by scopatz** [comment]
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

**Comment by hhslepicka** [comment]
@scopatz could we let Python setup deal with them? Otherwise I will need to keep it at two places...

**Comment by scopatz** [comment]
It won't work sadly, because conda needs to know how to relocate them

**Comment by hhslepicka** [comment]
Ok, I will send an update in a minute

**Comment by hhslepicka** [comment]
This error now is the missing libraries at the yum requirements for Qt. I will put a fix up in a bit.

**Comment by npavlovikj** [comment]
That sounds great @hhslepicka - thanks! Once we have the test working, we can go ahead and merge it.

**Comment by hhslepicka** [comment]
@npavlovikj and @scopatz thank you for the help getting this package it. Tests are passing and it looks ready to go.

**Comment by npavlovikj** [comment]
Everything looks great now @hhslepicka  - thank you for contributing and the prompt fixes!

**Pull request merged**